torchao/quantization/pt2e/_numeric_debugger.py

```diff
@@ -118,10 +118,14 @@ def _get_greatest_ancestor_node_source(node: Node) -> "NodeSource":
         return node_source
 
     def _is_node_in_original_graph(node: Node) -> bool:
+        # Handle guard nodes that don't have from_node metadata in newer PyTorch versions
```

Comment:
@tugsbayasgalan is this change expected? should we using non-strict mode or strict mode? https://github.com/pytorch/ao/blob/c96f2dd0c980ad100ea9e7c574dead8f19451955/test/quantization/pt2e/test_numeric_debugger.py#L51